pkg/services/rpcsrv/subscription_test.go

```diff
@@ -460,6 +462,46 @@ func TestFilteredBlockSubscriptions(t *testing.T) {
 	c.Close()
 }
 
+func TestHeaderOfAddedBlockSubscriptions(t *testing.T) {
+	// We can't fit this into TestFilteredSubscriptions, because it uses
+	// blocks as EOF events to wait for.
```

Comment:
Adjust this comment. I've managed to deal with `TestFilteredSubscriptions`, and it actually fails to invoke the `check` for headers because there's a single-node consensus (thus, a single primary), thus, no block events matching filter happen. If we're to add something like this to `TestFilteredSubscriptions`, then it works:
```
"header": {
			params: `["header_of_added_block", {"primary": 0, "since": 5}]`,
			check: func(t *testing.T, resp *neorpc.Notification) {
				rmap := resp.Payload[0].(map[string]any)
				require.Equal(t, neorpc.HeaderOfAddedBlockEventID, resp.Event)
				primary := rmap["primary"].(float64)
				require.Equal(t, 0, int(primary))
				index := rmap["index"].(float64)
				require.Less(t, 4, int(index))
			},
		},
``` 

But we'd better still have a separate `TestHeaderOfAddedBlockSubscriptions` as is, because we still need to check primary filters. So adjust this comment, it's not `because it uses blocks as EOF events to wait for.`